spanner/google/cloud/spanner_v1/database.py

```diff
@@ -245,6 +245,9 @@ def update_ddl(self, ddl_statements):
         See
         https://cloud.google.com/spanner/reference/rpc/google.spanner.admin.database.v1#google.spanner.admin.database.v1.DatabaseAdmin.UpdateDatabase
 
+        :type ddl_statements: list of DDL statements
```

Comment:
should be `Sequence[str]` (use type annotations for this)

Reply:
I changed it to `Sequence[str]` but for most of spanner `list of str` would be more of the norm, or even `list of string` in the file.  WDYT?

Reply:
for sphinx (old style) docstrings `list of str` is fine (never `list of string`, `string` is a module, not a type). For google/napoleon style docstrings prefer type annotation style.